packages/@uppy/tus/src/index.ts

```diff
@@ -238,7 +287,10 @@ export default class Tus extends BasePlugin {
       uploadOptions.onError = (err) => {
         this.uppy.log(err)
 
-        const xhr = err.originalRequest ? err.originalRequest.getUnderlyingObject() : null
+        const xhr =
+          err instanceof tus.DetailedError ?
```

Comment:
Can we avoid using `instanceof`, as discussed on Slack?

Reply:
I guess it depends what you mean by "equally safe", using `instanceof` tells you nothing on the value of `originalRequest`, and on the other hand you're going to miss actual instances of `tus.DetailedError` that were defined in another realm / with a different tus version. You can please TS either by using `as`, or by using a utility function.